docs/how-to/h-create-and-list-backups.md

```diff
@@ -0,0 +1,14 @@
+Creating and listing backups requires that you:
+* [Have a cluster with at least three nodes deployed](https://discourse.charmhub.io/t/charmed-mysql-tutorial-managing-units/TODO)
+* Access to S3 storage
+* [Have configured settings for S3 storage](https://discourse.charmhub.io/t/configuring-settings-for-s3/TODO)
+
+Once you have a three nodes cluster that has configurations set for S3 storage, check that Charmed MySQL is `active` and `idle` with `juju status`. Once Charmed MySQL is `active` and `idle`, you can create your first backup with the `create-backup` command:
+```
+juju run-action mysql/leader create-backup --wait
+```
+
+You can list your available, failed, and in progress backups by running the `list-backups` command:
```

Comment:
since we are synchronously taking backups, we are currently only listing the available backups. we can list failed backups, but that would require some refactoring of s3 helpers in the mysql charms

Reply:
I have removed requirements to have a cluster. On a single node, we can run `create-backup` on `leader`.
IMHO we should keep it AS IS.